File: The-Two-Sneaky-Numbers-of-Digitville.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getSneakyNumbers = function(nums) {
    let ans = [];
    nums.sort((a, b) => a - b);
    for(var i = 0; i < nums.length - 1; i++) {
        if(nums[i] == nums[i+1])
            ans.push(nums[i]);
    }
    return ans;
};

File: The-Two-Sneaky-Numbers-of-Digitville.cpp
class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& nums) {
        unordered_map<int, int> freq;
        vector<int> ans;
        for (auto num : nums) {
            freq[num]++;
            if (freq[num] > 1) {
                ans.push_back(num);
            }
        }
        return ans;
    }
};